helion/autotuner/config_spec.py

```diff
@@ -399,6 +429,36 @@ class RangeFlattenSpec(_OptionalBoolSpec):
     pass
 
 
+class StaticRangeSpec(_BlockIdItem):
+    def __init__(
+        self,
+        block_id: int,
+        is_static: bool,
+    ) -> None:
+        super().__init__([block_id])
+        self.is_static = is_static
```

Comment:
If `is_static` is False, don't create the StaticRangeSpec.

We shouldn't add config values with only a single choice.